package.json

```diff
@@ -6,7 +6,6 @@
     "node": ">=20.10.0"
   },
   "scripts": {
-    "prebuild": "npm audit --parseable | grep 'critical' && echo 'Critical vulnerabilities found. Please fix them before building. To do that run npm audit.' && exit 1 || true",
```

Comment:
Hi Nick. Why are you removing that?

Reply:
removed it when I was building with bun wince it wasn't being used, but I'll put it back as building with node now again